src/helpers/user-interaction.ts

```diff
@@ -134,36 +149,46 @@ function getQuestions(
   return questions;
 }
 
-export async function getAnswers(
-  program: commander.Command,
-  destination: string
-): Promise<Answers> {
+/**
+ * Prompt the user for mandatory options not set via CLI
+ *
+ * @param config Configuration data already set via CLI options
+ *
+ * @returns
+ * merged configuration options from CLI and user prompt
```

Comment:
it looks a bit odd  that the description is in the next line. Is that intended?

Reply:
I think it's cargo-cult motivated 😅 just saw it done that way somewhere else probably. Let me put them on same line

Reply:
Found the source: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/inquirer/index.d.ts

Reply:
thanks! Good to know you can do this :)

Reply:
Do you mean adding types or going full cargo cult? :)
Jokes aside, I'm taking the chance to improve my TS skills so if you see anything odd please let me know, I won't be offended but quite the opposite, right now is still "fake it till you make it"